### PR TITLE
feat(docker): healthcheck

### PR DESCRIPTION
**Pull request by JuniorJPDJ**
contains docker healthchecking as mentioned in #103

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -47,4 +47,5 @@ RUN python manage.py collectstatic --noinput && \
 # Launch
 USER appuser
 EXPOSE 8080
+HEALTHCHECK CMD bash -c 'wget -q -o /dev/null -O /dev/null --header "Host: ${ALLOWED_HOSTS%,*}" "http://127.0.0.1:$PORT/healthz/?format=json"'
```
Comment (wvffle) [inline Dockerfile:50]: Isn't `-q` acting same as `-o /dev/null`?

Also, wouldn't `curl -fso /dev/null -H "Host: ${ALLOWED_HOSTS%,*}` be better?

Comment (JuniorJPDJ) [reply Dockerfile:50]: This is about exit status code and I'm using busybox wget which is always available on alpine images.
I had trouble with making curl return error code like I'd like it to do.
It needs special flag for this and it returns other code than I want - masking it with ` || exit 1` would be needed which is not clean.
I'd like to stay with busybox wget.

> Isn't -q acting same as -o /dev/null?

To be honest: I've no idea. You may be right but I would need to check.

Comment (JuniorJPDJ) [reply Dockerfile:50]: It's not the same:

```
busybox wget -q http://googlesdfds.com
wget: bad address 'googlesdfds.com'
$ busybox wget -o /dev/null http://googlesdfds.com
<nothing>
```

Comment (JuniorJPDJ) [reply Dockerfile]: removed `-q`, thanks ;)

**Comment by JuniorJPDJ** [comment]
@milesmcc is anything still missing?

**Comment by milesmcc** [comment]
Pretty much looks good! Just to confirm, what's the difference between the two `o`/`O` flags (in `-o /dev/null -O /dev/null`)?

**Comment by JuniorJPDJ** [comment]
```
-O,  --output-document=FILE      write documents to FILE
[..]
-o,  --output-file=FILE          log messages to FILE
```

Which means -O is for output file downloaded using wget and -o is for wget errors/logged messages.

**Comment by milesmcc** [comment]
Cool, thanks

**Comment by djerfy** [comment]
Hi,

It's not good, my container doesn't start because healthcheck is ko:

```bash
[root] ~ # docker exec -ti shynet bash -c 'wget -o /dev/null -O /dev/null --header "Host: ${ALLOWED_HOSTS%,*}" "http://127.0.0.1:$PORT/healthz/?format=json"'; echo $?
1
```

With verbosity:

```bash
[root] ~ # docker exec -ti shynet bash -c 'wget --header "Host: ${ALLOWED_HOSTS%,*}" "http://127.0.0.1:$PORT/healthz/?format=json"'; echo $?
Connecting to 127.0.0.1:8080 (127.0.0.1:8080)
wget: server returned error: HTTP/1.1 400 Bad Request
1
```

Why the host header is important here?

**Comment by JuniorJPDJ** [comment]
and `ALLOWED_HOSTS` is being cut on first colon

`${ALLOWED_HOSTS%,*}` on `ALLOWED_HOSTS=domain1.com,domain2.com,domain3.com` should evaluate to `domain1.com`

Without setting HOST header it wasn't working at all.

I've two domains in allowed hosts and it works for me.
Could you modify healthcheck in docker-compose to show everything more verbose?

EDIT: Yes. It's my fault. It actually evaluates to this:
```
root@big-boy:~# ALLOWED_HOSTS=domain1.com,domain2.com,domain3.com
root@big-boy:~# echo ${ALLOWED_HOSTS%,*}
domain1.com,domain2.com
```

I've not checked more than 2 domains. Quick fix will be send as MR in a few minutes. Sorry ;)

**Comment by JuniorJPDJ** [comment]
Should be fixed now ;)
Could you please check my branch?

**Pull request merged**